### ISSUE TITLE
How can I get ISP model for your degrade model? 

### ISSUE THREAD

**Issue by yichuangzhang** [cszn/BSRGAN#41]

**Comment by sunyclj** [comment]
@yichuangzhang I have the same problem.Please,have you found the corresponding code?

**Comment by jarricd** [comment]
Read the paper. They cite a model in the paper.
https://github.com/timothybrooks/unprocessing - that's the repository for the reverse ISP model. You'll have to train in on your own.

**Comment by en1ero** [comment]
I don't think it is a trainable ISP model what you are looking for. I assume they just organized a series of image processing/unprocessing steps found in [unprocessing](https://github.com/timothybrooks/unprocessing) into PyTorch classes (instead of Tensorflow). These classes inherit from nn.Module in PyTorch and are grouped within a _isp_model_ class, allowing/forcing them to use the forward() method inside the degradation workflow.

So i guess this just enables PyTorch's functionalities for image processing.

**Comment by EugeneCh1a** [comment]
@cszn ，I have same problem in isp model
